app/models/user_option.rb

```diff
@@ -14,8 +14,7 @@ class UserOption < ActiveRecord::Base
   }
 
   self.ignored_columns = [
-    "disable_jump_reply", # Remove once 20210706091905 is promoted from post_deploy to regular migration
```

Comment:
I removed this because the migration has already been promoted to a regular migration.